package.json

```diff
@@ -79,9 +79,9 @@
     "mocha": "^3.2.0",
     "pretty-bytes": "^4.0.2",
     "qs": "^6.2.1",
-    "react": "^15.5.3",
-    "react-dom": "^15.5.3",
-    "react-transition-group": "^1.1.1",
+    "react": "16",
+    "react-addons-css-transition-group": "^15.6.0",
+    "react-dom": "16",
```

Comment:
Should the `react` and `react-dom` version be `16.0.0-rc.2`? `npm install` seems to be breaking

Reply:
Huh ... I used yarn to add it, so maybe npm doesn't understand that?

Reply:
Oh yeah, yarn works fine. npm seems to need the exact version. Or I'm on a older version of npm.

Reply:
Well, fails with npm 5 as well.

Reply:
I guess just manually install react@next and react-dom@next